views/templates/admin/tabs/listing/listing_export.tpl

```diff
@@ -0,0 +1,58 @@
+{**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *}
+
+<div id="module_list" class="panel panel-default col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0">
+    <div class="panel-heading">
+        <i class="material-icons">get_app</i>{l s='Export translations' mod='ps_translateyourmodule'}
+    </div>
+	<div class="panel-body">
+		<div class="row">
+			<table class="table table-striped">
+				<thead>
+					<tr>
+						<th colspan="2">{l s='Module name' mod='ps_translateyourmodule'}</th>
+						<th>{l s='Version' mod='ps_translateyourmodule'}</th>
+						<th class="table_select">{l s='See translations for' mod='ps_translateyourmodule'}</th>
+						<th colspan="2">{l s='Export translation XLSX' mod='ps_translateyourmodule'}</th>
+						<th>{l s='Export translation\'s folder' mod='ps_translateyourmodule'}</th>
+					</tr>
+				</thead>
+				<tbody>
```

Comment:
something wrong happened with indentation on this file

Reply:
Thanks